src/start/hello-xdp.md

```diff
@@ -4,7 +4,11 @@
 
 While there are myriad trace points to attach to and program types to write we should start somewhere simple.
 
-XDP (eXpress Data Path) programs permit our eBPF program to make decisions about packets that have been received on the interface to which our program is attached. To keep things simple, we'll build a very simplistic firewall to permit or deny traffic.
+XDP (eXpress Data Path) programs permit our eBPF program to make decisions about packets that have been sent out on the interface to which our program is attached. To keep things simple, we'll build a very simplistic firewall to permit or deny traffic.
```

Comment:
XDP is inbound only, I think this needs to be phrased better

Reply:
good catch! agreed.

Reply:
Any suggestions on how I could rephrase this to indicate that we are dropping on destination address but on the egress path?

Reply:
I think the previous text was fine?

But I also think that more generally the example could be clearer. Right now it makes assumptions about the ip addresses assigned to eth0, and I had never seen curl's `--interface` option before, why is it needed in this case?. It would probably be good to explain a bit more how the network interface is configured, maybe showing the output of `ip addr show eth0`